scripts/run-operator.sh

```diff
@@ -39,11 +39,13 @@ kubectl apply -n kafka -f https://strimzi.io/examples/latest/kafka/kafka-single-
 kubectl wait -n kafka kafka/my-cluster --for=condition=Ready --timeout=300s
 
 info "deleting example"
+set +e
```

Comment:
debug?

Reply:
was a fix, the script failed with a fresh minikube:

```
kubectl exited non-zero when we try to delete some resources, when k8s
has no knowledge of those CRDs. This command is intended to clean out
existing resources and CRDs from an established minikube, as well as
install into a fresh minikube. So we can tactically ignore those
failures.
```